bitnami/dataplatform-bp1/values.schema.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
I think this should not be necessary. Are you adding it because of certain reason?

Reply:
yes, we would like to have a form layout as well in kubeapps for user experience.